src/view/Listener.java

```diff
@@ -13,9 +15,13 @@
  * Created by Ahmed on 13-Aug-16.
  */
 public class Listener implements ActionListener {
-    private final MyFrame stpFrame;
     public static Record searchResult;
     public static boolean saveFlag = false;
+    private final MyFrame stpFrame;
+    public String currentDir = null;
+    public String selectedFile = null;
+    public String selectedFileAfterFix = null;
```

Comment:
and the user modified file is called ATTENDEES_CSV , we can change that later anyway